MakeGoldMaster.sh

```diff
@@ -7,7 +7,7 @@ cd $SCRIPTPATH
 
 mkdir -p IntegrationTests/GoldMaster
 
-for f in $(find Examples/*); do
+for f in $(find Examples/PodSpecs/*); do
```

Comment:
There are a lot of other things in the examples directory besides podspecs now so we have to be more specific

Reply:
Good catch 👍